src/main/views/privacy-policy.njk

```diff
@@ -116,6 +120,15 @@
         + 'Find out about this in our ' + internalLink('terms and conditions.', termsAndConditionsUri)) | safe }}
```

Comment:
There is no space between this line and the one above, so looks like "...shared with Google.Find out about..." in a browser. Just a very small fix.

Reply:
Thanks Rob, I have pushed the change.